docs/userGuide/syntax/dates.mbdf

```diff
@@ -0,0 +1,99 @@
+## Dates
+
+{% macro njcode(raw) %}<code>{<a/>{ {{ raw }} }}</code>{% endmacro %}
+{% macro njblock(raw) %}<code>{<a/>% {{ raw }} %}</code>{% endmacro %}
+
+**Markbind supports date formatting and simple calculations** as a Nunjucks [filter](https://mozilla.github.io/nunjucks/templating.html#filters).
+
+**Syntax:** {{ njcode('baseDate | date(format, daysToAdd)') }}
+
+<span id="main-example">
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+<box><span>
+{{ njblock('set base1 = "2019-08-12"') }}<br/>
+{{ njblock('set format1 = "DD MM YYYY"') }}<br/>
+{{ njblock('set format2 = "ddd DD/MM"') }}<br/>
+{{ njcode('base1 | date') }} `<!-- Mon 12 Aug -->`<br/>
+{{ njcode('base1 | date(format1)') }} `<!-- 12 08 2019 -->`<br/>
+{{ njcode('base1 | date(format1, 10)') }} `<!-- 22 08 2019 -->`<br/>
+{{ njcode('base1 | date(format2, 10)') }} `<!-- Thu 22/08 -->`<br/>
+</span></box>
+
+</span>
+
+<span id="output">
+Mon 12 Aug
+
+12 08 2019
+
+22 08 2019
+
+Thu 22/08
+</span>
+</include>
+
+<box type="info">
+
+The baseDate follows the format: `YYYY-MM-DD`
+
+It can be supplied using [page variables](../reusingContents.html#variables) for convenience.
```

Comment:
Can elaborate on this?

Reply:
The format portion or the page variables portion?

Reply:
page variable portion. May be give an example?